app/src/main/java/org/mozilla/focus/home/HomeFragment.java

```diff
@@ -330,7 +328,7 @@ private void initHomeBanner(Context context) {
     }
 
     private void initCouponBanner(Context context) {
-        initBanner(context, AppConfigWrapper.getCouponBannerRootConfig(), CURRENT_COUPON_BANNER_CONFIG, bannerConfigViewModel.getCouponConfig(), contentPanel::hideCouponBanner);
+        initBanner(context, AppConfigWrapper.getCouponBannerRootConfig(), CURRENT_COUPON_BANNER_CONFIG, bannerConfigViewModel.getCouponConfig(), null);
```

Comment:
consider removing the least parameter, the purpose of it was solely to delegate callback

Reply:
HomeBanner do need `hideHomeBannerProcedure`, so it's better to keep it.